browser-bunyan/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```

Comment:
They were different errors, but will try it again. Do you think it should be:

```
export * from "bunyan";
```
or
```
export * from "@types/bunyan";
```